homeassistant/components/profiler/__init__.py

```diff
@@ -74,7 +90,32 @@ async def _async_generate_profile(hass: HomeAssistant, call: ServiceCall):
     )
 
 
+async def _async_generate_memory_profile(hass: HomeAssistant, call: ServiceCall):
+    start_time = int(time.time() * 1000000)
+    hass.components.persistent_notification.async_create(
+        "The memory profile started. This notification will be updated when it is complete.",
```

Comment:
```suggestion
        "The memory profile has started. This notification will be updated when it is complete.",
```

Would be nice to fix my original bad grammar as well. 🤦

Reply:
Thanks 👍